File: test/cpp/LIM2Metrics/SizeMetrics/NUMPAR/NUMPAR.cpp

class A {
public:
	void foo(int a, int b) ;

};
	
void A::foo(int a, int b) {
	int c = 0;
	for (int i=0;i<=a;i++) {
		switch (b) {
			case 1 : 
			  c = 1;
			  break;
		    case 2 :
			  c = 10;
			  break;
			case 3 : 
			  c = 110;
			  break;
			default :
			  c = -1;
		}
	
	}

}

int main() {
   return 0;
}

File: test/java/LIM2Metrics/SizeMetrics/NUMPAR/NUMPAR.java

public class NUMPAR {
    public void draw(String s) {  // NUMPAR:1
        /*...*/
    }
    public void draw(int i) {  // NUMPAR:1
        /*...*/
    }
    public void draw(double f) {  // NUMPAR:1
        /*...*/
    }
    public void draw(int i, double f) {  // NUMPAR:2
        /*...*/
    }
	public int bar () {  // NUMPAR:0
		return 0;
	}
	public void foo(int ... a) {} // NUMPAR: 0
	public static void main(String [] args)
	{
		/*...*/
	}
}